sockeye/inference.py

```diff
@@ -1253,6 +1252,12 @@ def _beam_search(self,
                 models_output_layer_w.append(m.output_layer_w.take(vocab_slice_ids))
                 models_output_layer_b.append(m.output_layer_b.take(vocab_slice_ids))
 
+        # Select topK to use
+        if self.context == mx.cpu():
+            use_mxnet_topk = False
```

Comment:
I'd prefer one-line setting for this simple boolean: `use_mxnet_topk = self.context == mx.cpu()`